### DISCUSSION TITLE
Allow for line breaks in the Bio section

### DISCUSSION THREAD

**Discussion by haleygelato** [community/community#116072] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi! Could you please allow for line breaks in the Bio section of GitHub profiles? It's really frustrating to manually insert unicode characters to make it work. And sometimes, the result appears as expected on the web but looks strange on iOS due to differences in size.

TY!

**Comment by sujipark-fe** [comment]
The profile link also overflows the area. Please add `overflow: hidden` to `<a>`.

<hr>

# before
![image](https://github.com/user-attachments/assets/c9ab6a08-0445-446c-96bb-2a9a68c521f2)

<hr>

# after
![image](https://github.com/user-attachments/assets/9b0fd4a8-f3e7-4e03-bd5d-f6e8060bc70a)
